superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -111,17 +169,78 @@ class RefreshIntervalModal extends React.PureComponent<
         modalTitle={t('Refresh interval')}
         modalBody={
           <div>
-            <FormLabel>{t('Refresh frequency')}</FormLabel>
-            <Select
-              ariaLabel={t('Refresh interval')}
-              options={refreshIntervalOptions.map(option => ({
-                value: option[0],
-                label: t(option[1]),
-              }))}
-              value={refreshFrequency}
-              onChange={this.handleFrequencyChange}
-              sortComparator={propertyComparator('value')}
-            />
+            <div id="refresh_from_dropdown">
+              <FormLabel>
+                <b>{t('Refresh frequency')}</b>
+              </FormLabel>
+              <Select
+                ariaLabel={t('Refresh interval')}
+                options={this.createIntervalOptions(refreshIntervalOptions)}
+                value={refreshFrequency}
+                onChange={this.handleFrequencyChange}
+                sortComparator={propertyComparator('value')}
+              />
+            </div>
+            <div
+              style={{
```

Comment:
Not sure why we are still resorting to style visibility rather than rendering conditionally?

Reply:
that custom block visibility depends on whether the custom interval option is selected or not, what else can be done to render it conditionally?
from react i am changing the state of custom_block flag, and if it is true then the block gets rendered else not.

Reply:
What I am trying to say is that you should remove the element rather than hiding it with opacity.

See for example `{showRefreshWarning && (...)}` a few lines below.